lte/gateway/c/core/oai/tasks/amf/include/amf_client_servicer.h

```diff
@@ -26,6 +26,13 @@ extern "C" {
 #endif
 #include <memory>
 #include <string>
+#include <lte/protos/session_manager.grpc.pb.h>
```

Comment:
Use quotes for project headers
```suggestion
#include "lte/protos/session_manager.grpc.pb.h"
```

Reply:
review comment addressed